app/models/manageiq/providers/openstack/base_metrics_capture.rb

```diff
@@ -104,7 +104,7 @@ def find_meter_counters(metric_capture_module, resource_filter, metadata_filter,
         # fetch the list of resources and use the original_resource_id and type to find
         # the network interface's resource
         original_resource_id = "#{target.ems_ref}-tap#{port.ems_ref[0..10]}"
-        resources = @perf_ems.list_resources.body
+        resources = @perf_ems.list_resources('instance_network_interface').body
```

Comment:
It looks to me, that the ```instance_network_interface``` resource type is related to cloud provider only, but it is placed in file _base_metrics_capture.rb_ which is common for both infra and cloud providers.

I know that similar thing was merged few weeks ago (line 109), but I'd suggest have some separation* of cloud and infra code (metrics in cloud and infra can have different resources types AFAIK).

\* a condition or better a method or a different file.